set.mm

```diff
@@ -502199,6 +502199,64 @@ The sumset operation can be used for both group (additive) operations and
       JXLUUPUUOVEKYOGFXKVGXOXQXRXSUAAYPCYMEYFUBUCJHUWTUXAWNXTYA $.
   $}
 
+  ${
+    $d .0. x y z $.  $d B x y z $.  $d R x y z $.  $d U x y z $.
+    drngidl.b $e |- B = ( Base ` R ) $.
+    drngidl.z $e |- .0. = ( 0g ` R ) $.
+    drngidl.u $e |- U = ( LIdeal ` R ) $.
+    $( A ring is a division ring if and only if its only ideals are the zero
+       ideal and the unit ideal.  (Proposed by Gerard Lang, 13-Mar-2025.)
```

Comment:
@tirix  It should be:

> A **nonzero** ring is a division ring if and only if its only **left** ideals are the zero ideal and the unit ideal.

(and this is actually what you proved).

Without either of these two words, the comment is false: as you saw, the nonzero ring does have only two left ideals (actually, it's the only ring with at most one ideal); as for rings with zero divisors and exactly two ideals, take for instance $M_2(\mathbb{R})$.

Reply:
You're right, thanks! I will fix this in my next PR.